tests/integration/client/feedback/schemas/test_records.py

```diff
@@ -109,4 +125,53 @@ async def test_set_suggestions_deprecated(role: UserRole, db: "AsyncSession") ->
         for remote_record, factory_record in zip(remote_records, records):
             remote_record.set_suggestions(suggestions=[suggestion])
             await db.refresh(factory_record, attribute_names=["suggestions"])
-    assert all(record.suggestions == (suggestion,) for record in remote_dataset.records)
+    assert all(
+        isinstance(suggestion, RemoteSuggestionSchema)
+        for remote_record in remote_records
+        for suggestion in remote_record.suggestions
+    )
+
+
+@pytest.mark.parametrize("role", [UserRole.owner, UserRole.admin])
+@pytest.mark.asyncio
+async def test_delete_suggestions(role: UserRole, db: "AsyncSession") -> None:
+    field = await TextFieldFactory.create(required=True)
+    question = await TextQuestionFactory.create(required=True)
+    dataset = await DatasetFactory.create(fields=[field], questions=[question])
+    record = await RecordFactory.create(dataset=dataset)
+    await SuggestionFactory.create(record=record, question=question)
+    user = await UserFactory.create(role=role, workspaces=[dataset.workspace])
```

Comment:
I would recommend creating first the dataset and then creating the fields, records and questions passing the `dataset` argument, otherwise for each one a default dataset will be created.